pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -76,7 +76,7 @@
     public TenantInfo getTenantAdmin(
         @ApiParam(value = "The tenant name")
         @PathParam("tenant") String tenant) {
-        validateSuperUserAccess();
+        validateAdminAccessForTenant(tenant);
```

Comment:
This change is pretty smart - is there any documentation that needs to change as a result (from personal experience setting up AuthN/AuthZ is pretty hard outside of a standalone case). Thanks!

Reply:
Yes. We should document that. /cc @Huanli-Meng @Jennifer88huang

Reply:
ok, will let you know once the doc is updated. Thanks

Reply:
Hi @rdhabalia 

Since this code change affects both REST API and pulsar-admin docs, could you please help update the docs in this PR as below? (P.S. I've confirmed this w/ @tuteng) 
Many thanks~

- REST API
Go to [TenantsBase.java, Line 69](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java#L69), add `This operation requires admin privilege` after `Get the admin configuration for a given tenant.`

- pulsar-admin
Go to [CmdTenants.java, Line 43](https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTenants.java#L43), add `This operation requires admin privilege` after `Gets the configuration of a tenant.`

Reply:
Hi @rdhabalia, any thoughts on the docs?